doc/rst/usingchapel/prereqs.rst

```diff
@@ -21,10 +21,18 @@ for using Chapel:
 
   * You have access to ``gmake`` or a GNU-compatible version of ``make``.
 
-  * You have access to standard C and C++14 compilers. The C++14 support
-    is required for building the compiler itself. For GCC specifically,
-    GCC 5 or newer is required for C++14 support. Note that C11 support,
-    while not required, will enable faster atomic operations.
+  * You have access to standard C and C++ compilers.
+
+    * C++14 support is required for building the compiler and bundled
+      components, in addition to one of the following:
```

Comment:
I read "in addition to one of the following" as being about what C++14 is required for - as in "C++14 support is required for building the compiler and bundled components in addition to building GCC". Obviously this is not the right way to read it but can we reword it to make it clearer the following bullets are additional constraints on the C++ compiler?